server/paymentProviders/opencollective/host.js

```diff
@@ -16,6 +18,24 @@ paymentMethodProvider.getBalance = () => {
   return Promise.resolve(maxInteger);
 };
 
+paymentMethodProvider.createPlatformTipTransaction = async payload => {
+  const transaction = payload.transaction;
+  const TransactionGroup = transaction.TransactionGroup || uuid();
+  const donationTransaction = {
+    ...transaction,
+    amount: transaction.data.platformTipInHostCurrency,
```

Comment:
This was added recently

```suggestion
    amount: transaction.data.platformTipInHostCurrency,
    kind: TransactionKind.PLATFORM_TIP,
```

Reply:
Done 👍🏽